app/main.py

```diff
@@ -1,3 +1,74 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        if isinstance(other, (int, float)):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        if isinstance(other, (int, float)):
+            self.km += other
+            return self
+
+    def __mul__(self, other):
+        if isinstance(other, Distance):
+            return self.km * other.km
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        if isinstance(other, Distance):
+            return Distance(round(self.km / other.km, 2))
+        if isinstance(other, (int, float)):
+            return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other):
+        if isinstance(other, Distance):
+            return True if self.km < other.km else False
+
+        if isinstance(other, (int, float)):
+            return True if self.km < other else False
+
+    def __gt__(self, other):
+        if isinstance(other, Distance):
+            return True if self.km > other.km else False
+
+        if isinstance(other, (int, float)):
+            return True if self.km > other else False
```

Comment:
```suggestion
            return self.km > other
```
You can just return the value of the condition. It's already a boolean value